nextflow.config

```diff
@@ -169,14 +171,6 @@ params {
 }
 
 
-//--------
-// Work directory:
-//--------
-//This location stores the working temporary files and downloaded images
-//Note: If commented out - errors sometimes occur re failing to publish files (although files are published correctly)
-workDir = "$projectDir/work"
```

Comment:
This is a really bad idea to set like this. The standard way of running is to do:

```bash
nextflow run nf-core/dualrnaseq ...
```

Which will fetch the pipeline files from GitHub automatically and save them to `$NXF_HOME`. By default, this in `~/.nextflow`.

With the setting above, all intermediate pipeline files will be stored inside the pipeline source directory here. On many HPC systems, the user's home directory has a very small storage allocation and all analysis work should be done in project folders. So by default, this setting will break on those systems.

If this was added due to problems, the affected users should set work dir manually. It shouldn't be set in the pipeline code and it definitely shouldn't be inside the pipeline source code directory 😉

Reply:
I agree, but we had some errors during runtime which said files failed to be published - although the files were published correctly. This seemed to fix the issue.

Reply:
Yeah, I figured from the comment. Can you remember the details? We just need to fix in a different way I think..